apps/docs/src/components/mdx-components.tsx

```diff
@@ -37,6 +37,21 @@ const components = {
       {...props}
     />
   ),
+  code: ({ className, ...props }: React.HTMLAttributes<HTMLElement>) => (
```

Comment:
This causes some issues with the Command Component, see image:

Maybe we should update the component so it doesn't render a code tag inside.

Also, we should add the Code component to the UI package, and use it here 👍 

<img width="756" alt="image" src="https://github.com/medusajs/ui/assets/45367945/7fcde875-270d-4c75-8d9c-80f7edfd4f0c">

Reply:
Ah whoops, let me fix :D

Reply:
Made a `Code` component implementing a styled `code` tag. As for updating the Command, I'd leave it for now. It makes some assumptions when styling and spacing things, it somewhat _wants_ a `code` tag to surround the actual code you want in there. Let's revisit soon.

Reply:
I guess I _would_ rather be able to just do `\`some code\`` in MDX directly, rather than having to `<Code></Code>` inline for any piece of code 🤔